model/registry/db.yaml

```diff
@@ -38,12 +46,29 @@ groups:
           The name of the operation or command being executed.
         note: >
           It is RECOMMENDED to capture the value as provided by the application without attempting to do any case normalization.
+
+          For [homogeneous batch operations](/docs/database/database-spans.md#homogeneous-batches),
+          individual operations will all have the same operation name and so that operation name SHOULD be used,
+          prepended by `BATCH `.
+          For [heterogeneous batch operations](/docs/database/database-spans.md#heterogeneous-batches)
+          where individual operations are known to all have the same operation name, that operation name SHOULD be used,
+          prepended by `BATCH `.
+          For [heterogeneous batch operations](/docs/database/database-spans.md#heterogeneous-batches)
+          where individual operations are not known to all have the same operation name,
+          `db.operation.name` SHOULD be `BATCH` or some other database system specific term if more applicable.
```

Comment:
simplify: if known to have the same operation name, prepend by `BATCH `, otherwise capture `BATCH`